specs/aab-store and build/aab.md

```diff
@@ -0,0 +1,59 @@
+# App Center Stores and Build support for Android App Bundles
+	
+This proposal will cover a new feature to to add support for Android App Bundles in App Center.
+This feature will only cover support for .aab packages in the Stores service, and Build.
+	
+## Problem Statement
+	
+Google has introduced AAB and it's been the recommended method to upload packages to the Play Store.
+
+Google has also announced that By August 1, 2019, all apps that use native code must provide a 64-bit version in addition to the 32-bit version in order to publish an update. 
```

Comment:
Maybe make it clear that with `.aab` developers get the required 64-bit version.

Reply:
Added.